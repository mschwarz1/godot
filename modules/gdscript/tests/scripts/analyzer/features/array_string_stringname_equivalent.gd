func test():
	# Converted to String when initialized
	var string_array: Array[String] = [&"abc"]
	print(string_array)

	# Converted to StringName when initialized
	var stringname_array: Array[StringName] = ["abc"]
	print(stringname_array)
