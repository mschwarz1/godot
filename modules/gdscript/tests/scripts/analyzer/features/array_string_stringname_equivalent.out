GDTEST_OK
["abc"]
[&"abc"]
